proposals/hasCrossSiteAncestor.md

```diff
@@ -0,0 +1,116 @@
+# Proposal: Update Extensions to interact with Partitioned Cookies containing Cross-Site Ancestor Chain Bit values
+
+**Summary**
+
+Allow extensions to utilize a hasCrossSiteAncestor boolean value when interacting with partitioned cookies that include a cross-site ancestor chain bit in their partiton key.
+
+#### Background information: Description of Cross-site Ancestor Chain Bit in partitioned cookies
+
+A cross-site ancestor chain bit is a value that is set by the browser when a partitioned cookie is created. The value is used as part of the key in the partition key to determine access to a partitioned cookie. If the bit indicates true, it means the cookie has been set in a third party context.
+
+A third party context occurs when any parent frame, in the frame tree, is cross-site with the frame or subresource the cookie is being set on. If one site of the frame tree is third party, then all children frames of that frame are also third party and have a cross site ancestor. 
```

Comment:
This is not completely comprehensive. A few cases when the partition key may be cross-site not included here are:

- When the top-level frame makes a 3P subresource request
- When a 1P request is redirected to a 3P URL (even if it is subsequently redirected back to a 1P request after, that subsequent 1P request is now considered an ABA request)